functions/utils/recording_events.py

```diff
@@ -0,0 +1,233 @@
+import logging
+from datetime import datetime, timedelta
+import boto3
+from botocore.exceptions import ClientError
+from os import getenv as env
+from dotenv import load_dotenv
+from os.path import join, dirname
+from .common import TIMESTAMP_FORMAT
+from pytz import utc
+from itertools import chain
+
+logger = logging.getLogger()
+
+load_dotenv(join(dirname(__file__), "../.env"))
+
+RECORDING_EVENTS_TABLE = env("RECORDING_EVENTS_TABLE", None)
+
+START_EVENTS = ["recording.started", "recording.resumed"]
+END_EVENTS = ["recording.paused", "recording.stopped"]
+RECORDING_EVENTS = START_EVENTS + END_EVENTS
+
+# This was copied from status.py:
+# If RECORDING_EVENTS_TABLE is not set we assume the events table
+# is not being used. (For example tasks.py imports utils but doesn't
+# use the recording times table)
+recording_events_table = None
+if RECORDING_EVENTS_TABLE:
+    dynamodb = boto3.resource("dynamodb")
+    recording_events_table = dynamodb.Table(RECORDING_EVENTS_TABLE)
+
```

Comment:
It took me a bit but I think I finally got it. The `RECORDING_EVENTS_TABLE` variable will only be set in the context of a running lambda. Maybe update the comment to something more clear, like:

```
This value only ever comes from the lambda function's environment variable configuration. In a local environment, i.e., when running `tasks.py`, it will not be set.
```

Reply:
To be honest, I just copied it from Mona's code without really understanding why. The previous comment was not very helpful :) I will add the comment you suggested.